### DISCUSSION TITLE
Introduction to taxonomies

### DISCUSSION THREAD

**Discussion by mplabare** [biostars/biostar-handbook-issues#193] [General]
Multiple lines of code do not function. Starting with gzcat gi_taxid_nucl.dmp.gz | head produces this message:  Command 'gzcat' not found, did you mean:

  command 'bzcat' from deb bzip2 (1.0.8-2)
  command 'xzcat' from deb xz-utils (5.2.4-1ubuntu1)
  command 'grcat' from deb grc (1.11.3-1)
  command 'gscat' from deb scotch (6.0.9-1)
  command 'g3cat' from deb mgetty-fax (1.2.1-1)
  command 'zcat' from deb gzip (1.10-0ubuntu4)
  command 'zcat' from deb zutils (1.8-5build1)
  command 'lzcat' from deb lzma (9.22-2.1build1)
  command 'zzcat' from deb zziplib-bin (0.13.62-3.2ubuntu1)

Try: sudo apt install <deb name>

wget $URL

URL=https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz
 Produces this message: 
--2022-02-27 11:07:15--  https://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz
Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... failed: Temporary failure in name resolution.
wget: unable to resolve host address 'ftp.ncbi.nlm.nih.gov'

It goes on and on.

**Comment by ialbert** [comment]
It sounds like you system is not set up quite right if it cannot resolve a regular domain name (note the error message temporary failure in name resolution) 

Unfortunately there is a small inconsistency in how Unix works, what is called `gzcat` on a MacOS Unix is called `zcat` on Linux - I do make that note as well in the book, but perhaps not in that example.  Perhaps I should rewrite that command into two steps.

long story short use `zcat` instead of `gzcat`
